ui/packages/catalog-ui-search/src/main/webapp/js/model/QueryResponse.js

```diff
@@ -122,16 +122,21 @@ module.exports = Backbone.AssociatedModel.extend({
                         handled = true;
                         res.options = options;
                         switch (res.code) {
-                            case 400:
-                            case 404:
-                            case 500:
```

Comment:
These WebSocket responses don't have an HTTP status as their error code. It's one of the json rpc error codes that are used at the top of https://github.com/codice/ddf/blob/fd29e522b54d894045f3ec552db6119983a32005/catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/ws/JsonRpc.java#L34-L39

Reply:
These are application specific errors codes that reflect the ones in the spark apps and are passed on to the calling code.